OpenRA.Mods.Common/Traits/Buildings/BuildingUtils.cs

```diff
@@ -44,17 +52,18 @@ public static bool CanPlaceBuilding(this World world, string name, BuildingInfo
 				return true;
 
 			var res = world.WorldActor.TraitOrDefault<ResourceLayer>();
+			var actorInfo = world.Map.Rules.Actors[name];
 			return building.Tiles(topLeft).All(
 				t => world.Map.Contains(t) && (res == null || res.GetResource(t) == null) &&
-					world.IsCellBuildable(t, building, toIgnore));
+					world.IsCellBuildable(t, actorInfo, building, toIgnore));
 		}
 
 		public static IEnumerable<Pair<CPos, Actor>> GetLineBuildCells(World world, CPos location, string name, BuildingInfo bi)
 		{
 			var lbi = world.Map.Rules.Actors[name].TraitInfo<LineBuildInfo>();
 			var topLeft = location;	// 1x1 assumption!
 
-			if (world.IsCellBuildable(topLeft, bi))
+			if (world.IsCellBuildable(topLeft, null, bi))
```

Comment:
This should pass in the `ActorInfo` the same was as in `CanPlaceBuilding`.

Reply:
I've added that (commiting in a few) but was wondering what difference it makes, when IsCellBuildable gets the `ActorInfo` passed here